### PR TITLE
feat(AnalyticalTable): Add onRowExpandChange prop and placeholder loading

### PR DESCRIPTION
**Pull request by MarcusNotheis**
## BREAKING CHANGES
- Removed unused prop `cellHeight`, use `rowHeight` instead
- Removed unused prop `reactTableProps`
- Removed unused prop `getTableProps`
- Removed unused prop `getHeaderGroupsProps`
- Removed unused prop `getHeaderProps`
- Removed unused prop `getRowProps`
- Removed unused prop `getCellProps`

### CODE REVIEW

File: packages/main/src/components/AnalyticalTable/defaults/NoDataComponent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// const style = {
```
Comment (vbersch) [inline packages/main/src/components/AnalyticalTable/defaults/NoDataComponent/index.tsx]: do we need this?

File: packages/main/src/components/AnalyticalTable/demo/demo.stories.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { AnalyticalTable } from '@ui5/webcomponents-react/lib/AnalyticalTable';
 import { Button } from '@ui5/webcomponents-react/lib/Button';
 import { TextAlign } from '@ui5/webcomponents-react/lib/TextAlign';
 import { Title } from '@ui5/webcomponents-react/lib/Title';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (vbersch) [inline packages/main/src/components/AnalyticalTable/demo/demo.stories.tsx]: unused imports

File: packages/main/src/components/AnalyticalTable/hooks/useRowSelection.ts [hunk 3]
```diff
@@ -36,5 +36,9 @@ export const useRowSelection = (onRowSelected, selectedRowKeyProp) => {
     },
     [selectedRowPath, setSelectedRowPath, selectedRowKeyProp, prevSelectedRowKeyProp.current]
   );
+  useEffect(() => {
```
Comment (vbersch) [inline packages/main/src/components/AnalyticalTable/hooks/useRowSelection.ts]: was that out of desperation or is that effect really needed?

File: packages/main/src/components/AnalyticalTable/defaults/NoDataComponent/index.tsx (hunk 1 above)
Comment (MarcusNotheis) [reply packages/main/src/components/AnalyticalTable/defaults/NoDataComponent/index.tsx]: No :)

**Pull request merged**